src/components/Calculator.js

```diff
@@ -1,25 +1,30 @@
-/* eslint-disable react/prefer-stateless-function */
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useState } from 'react';
 import '../index.css';
 import Buttons from './Buttons';
 import Output from './Output';
+import calculate from '../logic/calculate';
 
-// eslint-disable react/prefer-stateless-function
-export default class Calculator extends React.Component {
-  render() {
-    return (
-      <div className="calculator-grid">
-        <Output
-          total={this.props.total}
-          operation={this.props.operation}
-          next={this.props.next}
-        />
-        <Buttons
-          clickHandler={this.props.clickHandler}
-        />
-      </div>
-    );
-  }
+export default function Calculator() {
```

Comment:
- [ ] Like the previous reviewer, It would be better to use ES6 syntax for functions(arrow function). 😉 
An example:
```
const Calculator = () => {
   content...
}

export default Calculator;
```

Reply:
Hello @ArthurGC ,

This is an old code you might be analyzing. Kindly [Take a look at this current updated file](https://github.com/Chimwemwe-127001/math-magician/blob/hooks/src/components/Calculator.js).
Hopefully, you checked and reviewed the latest commit.

Reply:
I am failing to understand how you got a different codebase than the current branch of this PR.

Kindly guide me, if I am missing a point regarding this comment.

> Cheers and HappyCoding :smiley:

Reply:
It seems like the file you reviewed is different from the current file on this brunch. 

Kindly check the link attached in the above comments, and revert post-confirmation.  This will help me know how to tackle the requested.

> Thanks once more :nerd_face:

Reply:
Actioned :heavy_check_mark: